playground/assets/css/css-url.css

```diff
@@ -27,13 +27,13 @@
 }
 
 .css-image-set-with-var {
-  --bg-img: url('../nested/asset.png');
+  --bg-img: url('/nested/asset.png');
```

Comment:
These URLs are ambiguous. In CSS, urls in variables are resolved from where the variable is _used_, not where it is defined. If `--bg-img` were referenced in a different file, it could resolve differently. Therefore, Lightning CSS disallows this and required absolute URLs.